Address Mohamed Boucadair's review (#32)
lamps-wg/x509-slhdsa: draft-ietf-lamps-x509-slhdsa.md

signature must come before the message. This is not the case for certificates and CRLs. Using
HashSLH-DSA reduces the size of the M' being held in memory.

> An SLH-DSA private key has a very large, but finite, number of signatures it can

Reviewer: ```suggestion
An SLH-DSA private key has a very large (2^64) number of signatures it can
```
I think it is more readable if we state the number here instead of (only) referring to the security considerations.

random numbers is difficult, and {{?RFC4086}} offers important guidance
in this area.

> Fault attacks can leak the SLH-DSA private signing key {{CMP2018}} and {{Ge2023}}.

Reviewer: ```suggestion
Fault attacks can lead to forgeries of message signatures {{CMP2018}} and {{Ge2023}}.
```
The cited papers state exactly this, not a private key leakage.
